_oasis

```diff
@@ -365,11 +369,19 @@ Library sema_test
   Install:        false
   Modules:        Test_ir
 
+Library serialization_test
+  Path:		  piqi/test
+  Build$:	  flag(tests) && flag(serialization)
+  CompiledObject: best
+  BuildDepends:	  bap, oUnit, bap.serialization
+  Install:	  false
+  Modules:	  Test_serialization
+
 Library benchmarks
   Path:           benchmarks
   Build$:         flag(tests) && flag(benchmarks)
   CompiledObject: best
-  BuildDepends:   bap, core, core_bench, threads
+  BuildDepends:   bap, core_bench, threads
```

Comment:
Ah, no, this is a bug in oasis, it adds flags to `_tags` file even if a feature is disabled. The core is needed if you specify `--with-benchmarks` option to `configure` script. So, please, add it back.